feat: new repeat elements script

1-2-3 opened the pull request.

zsviczian commented:
This is super cool!

zsviczian commented:
@1-2-3 : just to let you know, I added [ColorMaster](https://github.com/lbragile/ColorMaster#readme) to ExcalidrawAutomate. You can now use `ea.getCM(color: string)` to get a color master object. I also updated (largely simplified) Lighten color, Darken color and Repeat Elements. You can download the updated scripts from the script library and of course find them on GitHub.

1-2-3 commented:
> @1-2-3 : just to let you know, I added [ColorMaster](https://github.com/lbragile/ColorMaster#readme) to ExcalidrawAutomate. You can now use `ea.getCM(color: string)` to get a color master object. I also updated (largely simplified) Lighten color, Darken color and Repeat Elements. You can download the updated scripts from the script library and of course find them on GitHub.

Thank you very much for your work
